src/hooks/useValidation/useValidation.js

```diff
@@ -81,6 +83,26 @@ const useValidation = (context = {}) => {
     }, {});
   };
 
+  // remove field 001 error related to missing field for Bib records during create and derive,
+  // as this field is system generated and expected to be empty.
+  const removeError001MissingField = useCallback(formattedBEValidation => {
+    if (context.marcType !== MARC_TYPES.BIB || context.action === QUICK_MARC_ACTIONS.EDIT) {
+      return formattedBEValidation;
+    }
+
+    const issues = {
+      ...formattedBEValidation,
+      [MISSING_FIELD_ID]: formattedBEValidation[MISSING_FIELD_ID]?.filter(error => !error.tag.startsWith('001')) || [],
+    };
+
+    // Missed fields shouldn't be an empty array, so that the record can be saved on the first try if there are no other issues.
+    if (!issues[MISSING_FIELD_ID].length) {
+      delete issues[MISSING_FIELD_ID];
+    }

```

Comment:
It might be more clear to calculate MISSING_FIELD_ID in one step and then decide whether to add it in another. This code works, but adding it regardless of its value and then removing it if its value is empty is a little unusual.